frontend/src/component/feature/FeatureStrategy/FeatureStrategyForm/FeatureStrategyForm.tsx

```diff
@@ -350,19 +349,8 @@ export const FeatureStrategyForm = ({
             <StyledHeaderBox>
                 <StyledTitle>
                     {formatStrategyName(strategy.name || '')}
-                    <ConditionallyRender
-                        condition={
-                            !addEditStrategy &&
-                            strategy.name === 'flexibleRollout'
-                        }
-                        show={
-                            <Badge color='success' sx={{ marginLeft: '1rem' }}>
-                                {strategy.parameters?.rollout}%
-                            </Badge>
-                        }
-                    />
                 </StyledTitle>
-                {foundEnvironment && !addEditStrategy ? (
+                {foundEnvironment ? (
```

Comment:
This simplification is wrong. Because addEditStrategy is always true, this expression will always be false, so you can remove the entire check and the code inside it.